components/board/nanopi/board.go

```diff
@@ -0,0 +1,19 @@
+// Package nanopi implements a nanopi based board.
```

Comment:
[nit] tag as experimental please and add supported functionality of: GPIO pin outputs, I2C, SPI, Software PWM, and unsupported functionality of Digital Interrupts (?) and HardwarePWM (sure on this) don't work.

Reply:
Done